include/inviwo/core/common/inviwoapplication.h

```diff
@@ -143,23 +143,6 @@ class IVW_CORE_API InviwoApplication : public Singleton<InviwoApplication> {
 
     void registerModules(RuntimeModuleLoading, std::function<bool(std::string_view)> isEnabled);
 
-    /**
-     * Get the base path of the application.
-     * i.e. where the core data and modules folder and etc are.
-     */
-    std::filesystem::path getBasePath() const;
-
```

Comment:
More of a comment/note/issue: I think that we at some point could adjust the base path depending on the application. With the current implementation the application does not find the correct path during development _if_ the app is not in the inviwo source folder. In other words if one has a separate git repo for the app.